core/src/components/range/range.tsx

```diff
@@ -382,6 +403,7 @@ export class Range implements ComponentInterface {
     this.update(detail.currentX);
     this.pressedKnob = undefined;
 
+    this.emitValueChange();
```

Comment:
Why do we emit the value change here instead of in `updateValue`? The `this.update` call above should cause `updateValue` to get called.

Reply:
The browser implementation of range does not emit `change` for each move, only on release (or through keyboard navigation). By emitting in `updateValue`, `ionChange` would be emitted for each value change, even before release. 

This was done so that `ionChange` would be only emitted as a result of releasing the knob.